actiontext/lib/action_text/content.rb

```diff
@@ -97,8 +97,10 @@ def append_attachables(attachables)
 
     def render_attachments(**options, &block)
       content = fragment.replace(ActionText::Attachment.tag_name) do |node|
-        if node.key? "content"
+        if node.key?("content") && node["content"].present?
           node["content"] = sanitize_content_attachment(node["content"])
+        elsif node.key?("content")
+          node.remove_attribute("content")
```

Comment:
If the content after sanitation is empty we should probably remove it as well?
```suggestion
        if node.key?("content")
          sanitized_content = sanitize_content_attachment(node["content"])
          if sanitized_content.present?
            node["content"] = sanitized_content
          else
            node.remove_attribute("content")
```